upgrade_kafka_broker.yml

```diff
@@ -155,7 +159,7 @@
       import_tasks: tasks/wait_for_urp.yml
 
 - name: Kafka Broker Upgrade - Set Inter-Broker Protocol to Current Version
-  hosts: kafka_broker
+  hosts: kafka_broker_non_controllers,kafka_broker_controller
```

Comment:
there needs to be another call to the ordering tasks bc do not know the controller is the same